Palindrome.java

```diff
@@ -1,5 +1,34 @@
 class Palindrome {
 
+  // return true if sugus is a palindrome, not haribo
+  public boolean isStringPalindrome(String inputStr){
+
+      String j = new String("");
+
+      int l = 0;
```

Comment:
int l has a predictable value. inputStr.length could replace the increment in the for loop